a3p-integration/proposals/c:stake-bld/test-lib/index.js

```diff
@@ -0,0 +1,23 @@
+/* global setTimeout */
```

Comment:
the files in this directory are mostly from. I think we'll want to DRY it but I don't know when. I'm reluctant to have more exports from agoric-cli. I could see them going in ui-kit though. Or @agoric/contracts as support tools. I figure it's okay to duplicate for tests .

I put them in "test-lib" instead of "lib" in service of https://github.com/Agoric/agoric-3-proposals/issues/115

Reply:
> I could see them going in ui-kit though.

I lean that way.